opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -308,3 +312,281 @@ def test_collect(self):
         self.assertGreater(
             second_histogram.time_unix_nano, first_histogram.time_unix_nano
         )
+
+
+class TestConvertAggregationTemporality(TestCase):
+    """
+    Test aggregation temporality conversion algorithm
+    """
+
+    def test_mismatched_point_types(self):
+
+        current_point = Sum(
+            start_time_unix_nano=0,
+            time_unix_nano=0,
+            value=0,
+            aggregation_temporality=AggregationTemporality.DELTA,
+            is_monotonic=False,
+        )
+
+        with self.assertLogs(level=WARNING):
+            self.assertIs(
+                _convert_aggregation_temporality(
+                    Gauge(0, 0), current_point, AggregationTemporality.DELTA
```

Comment:
Can you update `Gauge(0, 0)` to use named arguments? Hard to remember what the positional arguments are

Reply:
fixed